File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cs
using System;

public class Solution 
{
    public string solution(string s) 
    {
        int i = s.Length - 1;
        
        string result = "";
        if(i%2 == 0)
        {
            result = s[i/2].ToString();
        }
        else
        {
            result = s[i/2].ToString();
            result += s[i/2 + 1].ToString();
        }
        return result;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) 
{
    string ans = "";
    if(s.length() % 2 == 1)
    {
       ans = s[(s.length() - 1) / 2];
    }
    else
    {
        ans += s[(s.length()) / 2 - 1];
        ans += s[(s.length()) / 2];
    }
    return ans;
}